doc/content/verification_and_validation/val-2a.md

```diff
@@ -1,58 +1,59 @@
+where $F$ is the implantation flux, $\sigma = 2.4 \times 10^{-9}$ m is the characteristic width of the normal distribution, and $\mu = 14 \times 10^{-9}$ m is the depth of the normal distribution from the upstream side. The comparison between the normal distribution from TMAP8 and piecewise function from TMAP4 is shown in [val-2a_normal_distribution]. The normal distribution has a similar distribution to the piecewise function, but the distribution profile is closer to the expected implantation profile.
 
-!media figures/val-2a_normal_distribution.png
+!media comparison_val-2a.py
+       image_name=val-2a_comparison_normal_distribution.png
        style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
        id=val-2a_normal_distribution
-       caption= Comparison between the normal distribution from TMAP8 and piecewise function from TMAP4. The area of both functions are normalized as 1.
+       caption= Comparison between the normal distribution from TMAP8 and piecewise function from TMAP4 for the source term due to deuterium ion beam implantation.
 
-The pressures on the upstream and downstream sides are close to the vacuum situation, and have less impact for the recommendation on two sides. Thus, TMAP8 ignore the impact of pressure on the boundary conditions to simplify the model. The recommendation is described as:
+The pressures on the upstream and downstream sides are close to vacuum pressures, and have only little impact for the recombination on both sides [!citep](longhurst1992verification,ambrosek2008verification). Thus, TMAP8 ignores the impact of pressure on the boundary conditions to simplify the model. The recombination is described asa simplified version of [eq:recommendation]:
 
 \begin{equation} \label{eq:recommendation_ignore_Pressure}
 J = A K_r C^2.
 \end{equation}
 
 ## Case and Model Parameters
 
-The beam flux on the upstream side of the sample during the experiment are presented in [val-2a_flux_and_pressure_TMAP4]. Other case and model parameters in TMAP8 are listed in [val-2a_set_up_values_TMAP4]. Some of the parameters are directly leveraged from [!cite](anderl1985tritium,longhurst1992verification,ambrosek2008verification), but others were adapted to better match the experimental data.
+The beam flux on the upstream side of the sample during the experiment is presented in [val-2a_flux_and_pressure_TMAP4], and only 75 % of the flux remain in the sample. Other case and model parameters used in TMAP8 are listed in [val-2a_set_up_values_TMAP4]. Some of the parameters are directly leveraged from [!cite](anderl1985tritium,longhurst1992verification,ambrosek2008verification), but others were adapted to better match the experimental data.
```

Comment:
```suggestion
The beam flux on the upstream side of the sample during the experiment is presented in [val-2a_flux_and_pressure_TMAP4], and only 75 % of the flux remain in the sample. Other case and model parameters used in TMAP8 are listed in [val-2a_set_up_values_TMAP4].
```
If you use values from TMAP4 and TMAP7, then this sentence can disappear. Otherwise, please specify what values are updated.